instagram-app/src/components/Login/Login.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+
+class Login extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            username: '',
+            password: ''
+        };
```

Comment:
To maintain clarity in naming, you might want to indicate in your state variables names that the variables hold inputText (perhaps `usernameInputText` and `passwordInputText` would be more helpful for this purpose).